hypha/apply/projects/templatetags/project_tags.py

```diff
@@ -147,3 +151,12 @@ def user_can_remove_supporting_documents(project, user):
     if user.is_apply_staff and project.status == COMMITTED:
         return True
     return False
+
+
+@register.simple_tag
+def user_can_take_actions(project, user):
```

Comment:
Can you add a docstring to this function to explain where it used. the `user_can_take_actionss` seems too generic to me here. in the `project_tags.py` context.